core/templates/dev/head/pages/collection_player/collection_player.html

```diff
@@ -103,15 +102,13 @@
 {% endblock navbar_breadcrumb %}
 
 {% block local_top_nav_options %}
-  <ul class="nav navbar-nav oppia-navbar-nav navbar-right" style="margin-right: 0px;">
-    {% if can_edit %}
-      <li>
-        <a ng-href="/collection_editor/create/{{collection_id}}" uib-tooltip="Edit" tooltip-placement="bottom" target="_blank">
-          <i class="material-icons">&#xE254;</i>
-          <span class="oppia-icon-accessibility-label">Edit</span>
-        </a>
-      </li>
-    {% endif %}
+  <ul class="nav navbar-nav oppia-navbar-nav navbar-right" style="margin-right: 0px;" ng-controller="CollectionLocalNav">
+    <li>
+      <a ng-href="/collection_editor/create/{{collection_id}}" uib-tooltip="Edit" tooltip-placement="bottom" target="_blank" ng-if="canEdit">
```

Comment:
Probably better to put the ng-if="canEdit" on the li (no point having an empty li element?)

Reply:
Done